dev/missing_dependency_tests/trivial_expectation.txt

```diff
@@ -1,4 +1,8 @@
 <<skip until matching line>>
 <<stderr>>
 <<skip until matching line>>
-Failed to load test harness\. +Are you missing a dependency on flutter_test\?
\ No newline at end of file
+Error: cannot run without a dependency on "package:flutter_test". Ensure the following lines are present in your pubspec.yaml:
```

Comment:
By checking the package config we know definitely there is a missing dependency. Added instructions on how to fix